src/App.css

```diff
@@ -7,7 +7,7 @@ html{
 body{
   margin: 0;
 }
-header{
+.Search{
```

Comment:
I would recommend to create its own css file for each component. Yes, we have a small project, but it makes sense to do it this way. Also consider using css-modules and SCSS.